pom.xml

```diff
@@ -73,6 +74,12 @@
       <version>${nukleus.kafka.spec.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.reaktivity</groupId>
+      <artifactId>nukleus-tcp.spec</artifactId>
+      <version>${nukleus.tcp.spec.version}</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
`nukleus-tcp.spec` dependency already pulled in by `nukleus-kafka.spec` dependency so no need to duplicate here, right?

Reply:
Yes except the tcp.spec dependency in nukleus-kafka.spec has test scope :( 
I have created PR https://github.com/reaktivity/nukleus-kafka.spec/pull/2 to fix that